runner/src/server/plugins/engine/components/UkAddressField.ts

```diff
@@ -235,6 +235,47 @@ export class UkAddressField extends AdapterFormComponent {
         );
     }
 
+    // arranging the address into an object
+    convertStringAnswersIntoObject(value: any) {
+        if (value !== "" && value !== undefined) {
```

Comment:
This seems to be almost an exact duplicate of the function below - is there not a way to reuse what's already there? That's a lot of logic to have in 2 places

Reply:
Yeah problem is in uk address field we have a different format that we used but inside the application we use this format so when we directly go to summary page it validate these so we have to convert string comma separated one into this exact format

Reply:
But it's the same logic just with different names in the object? Couldn't you have a function like `convertStringAnswersIntoObject(value: any, prefix:str="")` and then when you create the object:
```
const addressObject: any = {
            [`${prefix}addressLine1`]: "",
            [`${prefix}addressLine2`]: "",
            [`${prefix}town`]: "",
            [`${prefix}county`]: "",
            [`${prefix}postcode`]: "",
        };
```